code/game/objects/items/defibrillator.dm

```diff
@@ -1,7 +1,7 @@
-///The base healing number for a defibrillator. Kept global to be used by `check_revive()`.
+///The base healing number for a defibrillator. Needs to be global for `DEFIBRILLATOR_HEALING_TIMES_SKILL` to work outside of this file.
 #define DEFIBRILLATOR_BASE_HEALING_VALUE 8
-///The formula for healing with a defibrillator. Kept global to be used by `check_revive()`.
-#define DEFIBRILLATOR_HEALING_TIMES_SKILL(skill_input) (DEFIBRILLATOR_BASE_HEALING_VALUE * skill_input * 0.5)
+///The formula for healing with a defibrillator. If `skill_input` in this macro is less than `SKILL_MEDICAL_PRACTICED`, this will be 8.
+#define DEFIBRILLATOR_HEALING_TIMES_SKILL(skill_input) (skill_input < SKILL_MEDICAL_PRACTICED ? 8 : DEFIBRILLATOR_BASE_HEALING_VALUE * skill_input * 0.5)
```

Comment:
you'll probably want to move these defines to an actual define file now its being used in different files. Defines need to actually be defined before the code they're used in, so it breaks if someone tries to use this in a proceeding file.